install-deps.sh

```diff
@@ -144,6 +140,19 @@ function ensure_decent_cmake_on_ubuntu {
 	cmake
 }
 
+function ensure_decent_binutils_on_ubuntu {
+    local old=$(ld --version | head -n1 | grep -Po ' \K[0-9].*')
+    if dpkg --compare-versions $old ge $new; then
+        return
+    fi
+    install_pkg_on_ubuntu \
+	binutils \
+	7fa393306ed8b93019d225548474c0540b8928f7 \
```

Comment:
for those who are curious with this sha1, see https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=commit;h=7fa393306ed8b93019d225548474c0540b8928f7